### PR TITLE
Remove flakiness source from some tests.

### PR DESCRIPTION
**Pull request by rigelbm**

### CODE REVIEW

**Review by AndrewJSchofield — APPROVED** [review]

**Review by dimitarndimitrov — APPROVED** [review]
+1, because code-wise I can't see any problems (besides a couple of nits below), but can you please document the reasoning behind these changes, either in the commit, or in the PR description?

File: core/src/test/java/io/confluent/rest/ShutdownTest.java [hunk 1]
```diff
@@ -55,16 +58,17 @@ public void testGracefulShutdown() throws Exception {
...
+    Server server = app.createServer();
+    server.start();
 
-    RequestThread req = new RequestThread(config);
+    RequestThread req = new RequestThread(server.getURI());
     req.start();
     app.resource.requestProcessingStarted.await();
 
     StopThread stop = new StopThread(app);
     stop.start();
```
[the remark this answers is not in the document]
Comment (rigelbm) [reply core/src/test/java/io/confluent/rest/ShutdownTest.java:69]: It doesn't change anything.

File: core/src/test/java/io/confluent/rest/TestRestConfig.java [hunk 2]
```diff
@@ -34,11 +35,18 @@
     config = baseConfigDef();
   }
   public TestRestConfig() {
-    this(new Properties());
+    this(emptyMap());
```
[the remark this answers is not in the document]
Comment (rigelbm) [reply core/src/test/java/io/confluent/rest/TestRestConfig.java:38]: I prefer static import for methods that are self-descriptive by themselves :)

[the remark this answers is not in the document]
Comment (dimitarndimitrov) [reply core/src/test/java/io/confluent/rest/TestRestConfig.java:38]: You haven't seen multiple `emptyMap()` util methods on your classpath? You are one lucky guy!

**Comment by dimitarndimitrov** [comment]
@rigelbm Did you manage to address
> can you please document the reasoning behind these changes, either in the commit, or in the PR description?

**Pull request merged**